### PR TITLE
Add Own strategy and bare bone team page

### PR DESCRIPTION
**Pull request by malomarrec**
Initial, barebone strategy page.

### CODE REVIEW

File: content/departments/engineering/teams/own/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- For engineering support requests, customer questions, or anything else, @own-team on Slack.
+- @sourcegraph/batchers team or [own label](https://github.com/sourcegraph/sourcegraph/issues?q=is:issue+is:open+label:own) on GitHub.
+
+## Process
+
+- We prioritize our work in [our planning board](https://github.com/orgs/sourcegraph/projects/216). It is the responsibility of the PM and TL to keep this updated and correct.
+
+## Sprint planning
+
+Our two-week sprints start every other Tuesday.
```
Comment (eseliger) [inline content/departments/engineering/teams/own/index.md:20]: we currently do weekly sprints as we still move fast on decisions

File: content/strategy-goals/strategy/own/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Quick links
+
+- [Engineering strategy](../../../departments/engineering/index.md#product-vision-and-strategy)
+- [Own](../../../departments/engineering/teams/own/index.md)
+- [Product Marketing Brief](https://docs.google.com/document/d/1597R1KNoyrU79twn0xoBBMxexnRRGe2I1VQA4mxtR-I/edit#heading=h.a9uhc0ymvnxt)
+
+## Vision
+
+In 3 years, nobody will have to worry about codeowners being out of date. Devs will be able to quickly understand who owns what code and service, and how all the pieces fit together, and collaboration will be so much easier. The People Graph and the Source Graph will be merged.
```
Comment (cbart) [inline content/strategy-goals/strategy/own/index.md]:
```suggestion
In 3 years, nobody will have to worry about CODEOWNERS being out of date. Devs will be able to quickly understand who owns what code and service, and how all the pieces fit together, and collaboration will be so much easier. The People Graph and the Source Graph will be merged.
```

File: content/strategy-goals/strategy/own/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Allow developers to quickly answer questions about relationships between code, and the persons who build or interact with it. Make all the workflows that require navigating the org chart to improve or fix code easy. Reduce noise, automate boring tasks, kill useless emails.
+
+## Strategy
+
+Own is an emerging product, that we plan to release as an MVP at Starship in March 2023. Our product strategy is relatively bare-bone at this stage:
+
+- First, we'll lay some foundatations. Sourcegraph Own will ingest existing ownership data (eg. from CODEOWNERS) and provide powerful ways to use it that neither CODEOWNERS nor in-house systems provide, building on the power of Sourcegraph.
+  - What code does alice own that has the AuthParser class? Search for `AuthParser file:has.owner(@alice)`.
+  - We need to remove AuthParser from the codebase. Who owns code that uses it? Search for `AuthParser select:owner`, get a list of all the owners, quickly contact them.
+  - Search for files without owner
```
Comment (cbart) [inline content/strategy-goals/strategy/own/index.md:28]:
```suggestion
  - Search for files without owner `-file:has.owner()`.
```

File: content/strategy-goals/strategy/own/index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+- Second, we'll make ownership 10x easier and 10x more valuable.
+  - We'll make it so ownership can be inferred automatically, and be evegreen, removing the need for maintenace and solving a major pain point. We'll make it easy to answer both "who's accountable for this code" and "who owns this code", and provide escalation paths.
```
Comment (cbart) [inline content/strategy-goals/strategy/own/index.md]:
```suggestion
  - We'll make it so ownership can be inferred automatically, and be evergreen, removing the need for maintenance and solving a major pain point. We'll make it easy to answer both "who's accountable for this code" and "who owns this code", and provide escalation paths.
```

**Review by cbart — APPROVED** [review]
Just noted some typos. Looks good.

File: content/strategy-goals/strategy/own/index.md (hunk 4 above)
Comment (malomarrec) [reply content/strategy-goals/strategy/own/index.md]: ty!

File: content/departments/engineering/teams/own/index.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+## Strategy
+
+- Vision, mission and strategy: [Own strategy](../../../../strategy-goals/strategy/own/index.md)
+- Key metrics: TODO after launch!
+
+## Contact
+
+- For general questions or concerns, #ask-own channel or @own-team on Slack.
+- For engineering support requests, customer questions, or anything else, @own-team on Slack.
+- @sourcegraph/batchers team or [own label](https://github.com/sourcegraph/sourcegraph/issues?q=is:issue+is:open+label:own) on GitHub.
```
Comment (limitedmage) [inline content/departments/engineering/teams/own/index.md]: nit: update to the new GitHub team :)

**Review by limitedmage — APPROVED** [review]

File: content/departments/engineering/teams/own/index.md (hunk 5 above)
Comment (malomarrec) [reply content/departments/engineering/teams/own/index.md]: Good catch, thank you!

File: content/strategy-goals/strategy/own/index.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+## Strategy
+
+Own is an emerging product, that we plan to release as an MVP at Starship in March 2023. Our product strategy is relatively bare-bone at this stage:
+
+- First, we'll lay some foundatations. Sourcegraph Own will ingest existing ownership data (eg. from CODEOWNERS) and provide powerful ways to use it that neither CODEOWNERS nor in-house systems provide, building on the power of Sourcegraph.
+  - What code does alice own that has the AuthParser class? Search for `AuthParser file:has.owner(@alice)`.
+  - We need to remove AuthParser from the codebase. Who owns code that uses it? Search for `AuthParser select:owner`, get a list of all the owners, quickly contact them.
+  - Search for files without owner
+- Second, we'll make ownership 10x easier and 10x more valuable.
```
Comment (eseliger) [inline content/strategy-goals/strategy/own/index.md:29]: this and beyond is not march 2023, correct?

**Review by eseliger — COMMENTED** [review]
sorry for being late, got side-tracked by merge demos but lgtm, thanks for doing this!

File: content/strategy-goals/strategy/own/index.md (hunk 6 above)
Comment (malomarrec) [reply content/strategy-goals/strategy/own/index.md:29]: This is strategy, so is 1 year+ ahead (less would be good, too!)

**Pull request merged**